src/persistence/settings.cpp

```diff
@@ -1175,6 +1177,19 @@ void Settings::setCamVideoRes(QSize newValue)
     camVideoRes = newValue;
 }
 
+unsigned short Settings::getCamVideoFPS() const
+{
+    QMutexLocker locker{&bigLock};
```

Comment:
@agilob True! Thanks! I strike through the "bullshit" I wrote above :innocent:. Like stated by @TheSpiritXIII in another context, this can be solved by RAII -> in a new PR.